src/core/reducers/addons.js

```diff
@@ -194,7 +194,7 @@ export function createInternalAddon(apiAddon: ExternalAddonType): AddonType {
     // TODO: remove this if possible. This is used by core/installAddon
     // and DiscoPane components which do camel case conversions for
     // some historic reason.
-    iconUrl: apiAddon.icon_url,
+  
```

Comment:
I don't think this is enough. The `iconUrl` property is used in the code as described in the code comment above.

Reply:
okk @willdurand !

Reply:
@willdurand I have a question. i have to refractor iconUrl to icon_url or remove it!

Reply:
it is about removing iconUrl but this property is unfortunately used in some places.